### DISCUSSION TITLE
Error in when deployed in Nextjs api function - Error: Cannot find module '/var/task/node_modules/noblox.js/lib/cache'

### DISCUSSION THREAD

**Discussion by jeremytenjo** [noblox/noblox.js#771] [Q&A]
Hi, does anybody know how to fix the this error when Nobloxjs used in a Nextjs api function ?

`Error: Cannot find module '/var/task/node_modules/noblox.js/lib/cache'`

![image](https://github.com/noblox/noblox.js/assets/25231946/4c699bf7-8f99-483f-b51c-6f2bec8ebdf9)

`"noblox.js": "4.15.0"`

Thank you!

**Comment by Neztore** [comment]
If this is still an issue please re-open
